index.html

```diff
@@ -14,183 +14,7 @@
 
     <title>Frontend Mentor | Fylo landing page with two column layout</title>
     <link rel="stylesheet" href="style.css" />
-    <style>
-      .attribution {
-        font-size: 11px;
-        text-align: center;
-      }
-      .attribution a {
-        color: hsl(228, 45%, 44%);
-      }
-    </style>
   </head>
   <body>
-    <header>
-      <div class="icon">
-        <img src="./images/logo.svg" alt="Logo" />
-      </div>
-      <nav class="navbar">
-        <div class="hamburger-icon">
-          <img src="./images/menu-hamburger.svg" alt="" onclick="openMenu()" />
-        </div>
-        <div class="menu" id="menu">
-          <ul>
-            <li><a href="#">Features</a></li>
-            <li><a href="#">Team</a></li>
-            <li><a href="#">Sign In</a></li>
-          </ul>
-        </div>
-      </nav>
-    </header>
-
-    <main>
-      <section class="hero">
-        <div class="hero-img">
-          <img src="/images/illustration-1.svg" alt="illustration-1" />
```

Comment:
This image isn't showing for me. To ensure assets are visible to all, use relative paths: this means adding a dot in front of the first slash, like you've done on line 72